### PR TITLE
Limit width of code-blocks to prevent wraparound of TOC

### PR DESCRIPTION
**Pull request by shorgi**
Fixes:
- #1210 

The specific goal of this PR is to prevent code blocks with long, unbroken strings from forcing the right-nav TOC off the visible screen.

Previews of pages that previously had code blocks so wide that they pushed the right-nav TOC to the bottom:
- https://deploy-preview-1224--redpanda-documentation.netlify.app/docs/manage/security/encryption/
- https://deploy-preview-1224--redpanda-documentation.netlify.app/docs/develop/http-proxy/

### CODE REVIEW

**Comment by Deflaimun** [comment]
Still happening
image

File: src/css/custom.css [hunk 1]
```diff
@@ -149,6 +136,13 @@ div .admonition-content > ul > li > a {
   display: block;
   overflow-x: auto;
   white-space: nowrap;
+  }  
+}
+
+/* Prevent right-nav TOC from appearing below footer */
+@media screen and (max-width: 1008px)  {
```
Comment (Deflaimun) [inline src/css/custom.css:143]: Why are we limiting to max screen size ?

**Comment by shorgi** [comment]
> Still happening image

Yes, this retains the behavior of hiding both the left-nav and right-nav when the screen is narrow. Showing the right-nav TOC for narrow widths would make it appear for mobile; that's not Docusaurus's default behavior, and I assume it's not ours either.

**Comment by micheleRP** [comment]
Sorry @Deflaimun I just saw this. The Encryption page will get fixed with https://deploy-preview-1148--redpanda-documentation.netlify.app/docs/manage/security/encryption/ (since we don't link to that gist anymore)

**Comment by shorgi** [comment]
Updated the css to force the right-nav TOC to always appear at the same time as the left-nav.

**Comment by Deflaimun** [comment]
Looks like it's working. Great job! 
image

File: src/css/custom.css [hunk 2]
```diff
@@ -144,14 +131,26 @@ div .admonition-content > ul > li > a {
   margin-top: 0;
 }
 
+/* Don't wrap right-nav TOC in main-doc container when screen width > 996px */ 
+[class*=docMainContainer] .container .row {
```
Comment (Deflaimun) [inline src/css/custom.css:135]: I like this !

**Review by Deflaimun — APPROVED** [review]

**Comment by Deflaimun** [comment]
LGTM!

**Pull request merged**